libs/common/src/lib/localization/localization.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from "@nestjs/common";
+import { i18nTranslation, LocalizationKey } from "@terramatch-microservices/database/entities";
+import { i18nItem } from "@terramatch-microservices/database/entities/i18n-item.entity";
+import { Op } from "sequelize";
+
+
+@Injectable()
+export class LocalizationService {
+
+  async getLocalizationKeys(keys: string[]): Promise<LocalizationKey[]> {
+    return await LocalizationKey.findAll({ where: { key: { [Op.in]: keys } } });
+  }
+
+  private async getItemI18n(value: string): Promise<i18nItem | null> {
+    return await i18nItem.findOne({
+      where: {
+        [Op.or]: [
+          { shortValue: value },
+          { longValue: value }
+        ]
+      }
+    });
+  }
+
+  private async getTranslateItem(itemId: number, locale: string): Promise<i18nTranslation | null> {
+    return await i18nTranslation.findOne({where: { i18nItemId: itemId, language: locale}});
```

Comment:
There are several cases of our `prettier` styles not being applied in this PR. Please set that up locally and apply it. Your editor should be able to automatically have prettier run on file save. Some time soon I'll get our linter set up to fail on things like this (the spacing around your curly braces is missing). I don't really understand why the out of the box NX / NestJS setup doesn't enforce that already.